DEVELOPMENT.md

```diff
@@ -17,44 +16,48 @@ https://docs.expo.dev/get-started/set-up-your-environment/ helps you to install
 ## First time setup
 
 ```sh
-corepack enable   # prerequisite to install yarn automatically on next step
```

Comment:
We still need to enable corepack on first run, it is not enabled by default

Reply:
why? Is npm not installed by default anymore?

Reply:
npm should be installed with every node installation, but corepack still needs to be enabled, as far as I know

Reply:
That's the first time I hear that in my years of javascript development. This instance was the first time I ever used corepack.

Reply:
I have also never used corepack before, but since it's here already, let's just keep it

Reply:
I See no reason to include it unless we use an alternative package manager like yarn or pnpm. But If you want to keep it we can also do that, I don't have strong opinions about this and would like to get the pr merged soon.